clerk/organizations.go

```diff
@@ -198,6 +205,16 @@ func (s *OrganizationsService) ListMemberships(params ListOrganizationMembership
 	endpoint := fmt.Sprintf("%s/%s/memberships", OrganizationsUrl, params.OrganizationID)
 	req, _ := s.client.NewRequest(http.MethodGet, endpoint)
 
+	s.client.Users().addUserSearchParamsToRequest(req, ListAllUsersParams{
```

Comment:
Not feeling that confident to reuse this as it is, as it's user specific. I would prefer to refactor this helper a bit to be generic. Then we can also use it for this operation as well

Reply:
I had the same feeling, but i said to give it a go! I will create a new function inside organizations.

Reply:
I had the same feeling for this, do you think it will be better to create a new function for the organizations? I was thinking also to put a new function in a shared file so both of them have access. Do we have this option here?